_gen/index.md

```diff
@@ -30,11 +30,11 @@ For package management we'll use jsonnet-bundler, please
 > don't know how Kubernetes works, this isn't a requirement for understanding the Jsonnet
 > examples.
 
-1. [Write an extensible library](lesson1.md)
-1. [Understanding Package management](lesson2.md)
-1. [Exercise: rewrite a library with `k8s-libsonnet`](lesson3.md)
-1. [Further developing libraries](lesson4.md)
-1. [Providing documentation with Docsonnet](lesson5.md)
+1. [Introduction exercise](lesson1.md)
+1. [Package management](lesson2.md)
+1. [Refactoring exercise](lesson3.md)
+1. [Wrapping libraries](lesson4.md)
+1. [Documentation](lesson5.md)
```

Comment:
make this `Documenting` instead of `Documentation. That will tell me it is about how I *do* something.

Reply:
or better: `Documenting Code`.